src/biotite/sequence/graphics/alignment.py

```diff
@@ -344,7 +344,117 @@ def get_color(self, alignment, column_i, seq_i):
         code = alignment.sequences[seq_i].code[index]
         return self._colors[code]
 
+    
+class ArrayPlotter(LetterPlotter):
+    '''
+    This :class:`SymbolPlotter` quantitatively decorates sequences alignments, with molecular
+    recognition data obtained from e.g. microarrays. Symbols are visualized as characters 
+    on a colored background box. The color of a given box represents the recognition 
+    signal. The intensity of the color, is proportional to the strenght of the 
+    recognition.
+    
+    Parameters
+    ----------
+    axes : Axes
+        A Matplotlib axes, that is used as plotting area.
+    fl_score : numpy.ndarray 
+        The ndarray to store recognition values corresponding to the score residues.
+        By default, the normalized score is 1 for maximum recognition
+        and 0 for non-recognition (no color).
+    color_symbols : bool, optional
+        If true, the symbols themselves are colored.
+        If false, the symbols are black, and the boxes behind the
+        symbols are colored.
+    font_size : float, optional
+        Font size of the sequence symbols.
+    font_param : dict, optional
+        Additional parameters that is given to the
+        :class:`matplotlib.Text` instance of each symbol.
+    
+    '''
+    def __init__(self, axes, fl_score, color_symbols=False,
+                 font_size=None, font_param=None):
 
+        super().__init__(axes, color_symbols, font_size, font_param)
+        self.fl_score = fl_score       
+        self._cmap = self._generate_colormap(colors["dimorange"],
```

Comment:
Yes, I will leave it out from `ArrayPlotter` at the moment. It should not be required by `plot_alignment_array()`

Reply:
But the following lines call set_color:

https://github.com/biotite-dev/biotite/blob/b86594b110d46717f2657a13951403d12ce98f78/src/biotite/sequence/graphics/alignment.py#L1003-L1004

I tested setting the `color` parameter in `plot_alignment_array()` in your example and it would lead to:

`AttributeError: 'ArrayPlotter' object has no attribute 'set_color'`

Reply:
You are right. Initially I wanted to enable set_color() on ArrayPlotter, but dropped it later on. Sorry, those lines were vestiges from the past. I just deleted those lines.